src/js/showScores.js

```diff
@@ -0,0 +1,16 @@
+import { dataScores } from './dataScores.js';
+
+const scoresContainer = document.querySelector('.scores__ul');
+
+const showScores = () => {
+  scoresContainer.innerHTML = '';
+  dataScores.forEach((score) => {
+    const scoreLi = document.createElement('li');
+    scoreLi.className = 'scores__li';
+    scoreLi.innerHTML = `${score.name}: ${score.score}`;
+    scoresContainer.appendChild(scoreLi);
+  });
+};
+
+/* eslint-disable */
```

Comment:
- [ ] Same here, it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉